app/models/order_detail.rb

```diff
@@ -171,7 +171,6 @@ def self.in_dispute
     where("dispute_at IS NOT NULL")
       .where(dispute_resolved_at: nil)
       .where("order_details.state != ?", "canceled")
-      .order("dispute_at")
```

Comment:
Does this ordering still get applied where it's needed (i.e. the controller)?

Reply:
Added that back in.